test/Microsoft.NET.TestFramework/TestAsset.cs

```diff
@@ -234,6 +234,20 @@ private bool IsInBinOrObjFolder(string path)
                 {
                     yield return Tuple.Create(dependency.Attribute("id").Value, dependency.Attribute("version").Value);
                 }
+
+                yield return Tuple.Create("System.Diagnostics.TraceSource", "4.0.0");
+            }
+        }
+
+        public static string ConflictResolutionTestMethod
```

Comment:
This seems like a pretty specific thing to be on the general `TestAsset`.  The above method went from something that was more general "netstandard1.3. stuff", to "ConflictResolution"-specific.

Is there a better place for this conflict resolution stuff than on the general `TestAsset`?